### DISCUSSION TITLE
Facebook

### DISCUSSION THREAD

**Discussion by kuchingneko28** [catppuccin/catppuccin#2620] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://facebook.com/
https://web.facebook.com/

### Submission Guidelines

- [x] I have read and followed Catppuccin's submission guidelines.

**Comment by WalkQuackBack** [comment]
There's a PR open for this: https://github.com/catppuccin/userstyles/pull/1845
